audits/performance/first-meaningful-paint.js

```diff
@@ -71,17 +67,21 @@ class FirstMeaningfulPaint extends Audit {
           score = Math.max(0, score);
 
           return {
-            duration: `${fmp.duration.toFixed(2)}ms`,
+            duration: `${firstMeaningfulPaint.toFixed(2)}ms`,
             score: Math.round(score)
           };
-        }, _ => {
+        }, err => {
```

Comment:
i really dislike the , style pipeline approach here... it makes it even easier to forget when you are in a catch and when you are not. Instead i prefer the explicit `.catch()` pipelining approah